src/components/CardComponent.css

```diff
@@ -0,0 +1,47 @@
+:root {
+    --clr-gray-light: #b0bec5;
+  }
+.card {
+    overflow: hidden;
+    box-shadow: 0px 2px 8px 0px var(--clr-gray-light);
+    background-color: white;
+    text-align: center;
+    border-radius: 1rem;
+    position: relative;
+    width: 280px;
+    margin-bottom: 1rem;
+    margin-left: 1rem;
+  }
+
+.designation {
+    color: grey;
+    font-size: 18px;
+}
+.linkedin-info{
+    margin: 24px 0;
+}
+
+  
+.banner-img {
+position: absolute;
+background-image: url(https://images.unsplash.com/photo-1582727657635-c771002bdada?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjE0NTg5fQ);
```

Comment:
try to add this from the JS code as it becomes much easier to track. some times it's easy to lose track of URLs mentioned in css. Though if you use scss, sass, you can do this by creating variables in css itself